floris/heterogeneous_map.py

```diff
@@ -39,6 +41,7 @@ def __init__(
         x: NDArrayFloat,
         y: NDArrayFloat,
         speed_multipliers: NDArrayFloat,
+        z: NDArrayFloat = None,
```

Comment:
Is the reason this is after `speed_multipliers` to allow it to take a default (while `speed_multipliers` has no default)?

What if we reorder it (to `x`, `y`, `z`, `speed_multipliers`) and give `speed_multipliers` a default of 1?

Reply:
Yeah, that's it.  Admit it isn't a perfect solution but I think it is somehow correct that speed_multipliers has no default, but willing to discuss more

Reply:
Yeah, maybe that's right. Ok, hopefully people just use keyword arguments then and then can specify them in whatever order they please.